Submitted By: Ken Moffat <ken at linuxfromscratch dot org>
Date: 2016-03-20
Initial Package Version: 0.12
Upstream Status: Unknown
Origin: Found at debian
Description: Disable broken tests
Author: Jan Wagner <waja@cyconet.org>
Last-Update: 2015-10-11

--- a/test.pl
+++ b/test.pl
@@ -1,5 +1,5 @@
 use Test;
-BEGIN { plan tests => 4 };
+BEGIN { plan tests => 3 };
 use Data::Uniqid qw ( suniqid uniqid luniqid ) ;
 
 print "suniqid -> " . suniqid . "\n";
@@ -27,11 +27,11 @@
 }
 if ($err>0) {;ok(0);} else {;ok(1);}
 
-use Sys::Hostname;
-my($ia,$ib,$ic,$id)=unpack("C4", (gethostbyname(hostname()))[4]);
-
-if ($ia=~/\d+/ && $ia>0 && $ib=~/\d+/ && $ib>0 && 
-    $ic=~/\d+/ && $ic>0 && $id=~/\d+/ && $id>0 ) {;ok(1);}
-else {;ok(0);}
+#use Sys::Hostname;
+#my($ia,$ib,$ic,$id)=unpack("C4", (gethostbyname(hostname()))[4]);
+#
+#if ($ia=~/\d+/ && $ia>0 && $ib=~/\d+/ && $ib>0 && 
+#    $ic=~/\d+/ && $ic>0 && $id=~/\d+/ && $id>0 ) {;ok(1);}
+#else {;ok(0);}
 
 
